packages/changed-elements-react/src/widgets/comparisonJobWidget/common/versionCompareV2WidgetUtils.ts

```diff
@@ -23,9 +23,13 @@ export type ManagerStartComparisonV2Args = {
   iModelsClient: IModelsClient;
 };
 
+const handleStopCompare = async (): Promise<void> => {
+  await VersionCompare.manager?.stopComparison();
+};
+
 export const runManagerStartComparisonV2 = async (args: ManagerStartComparisonV2Args) => {
   if (VersionCompare.manager?.isComparing) {
-    return;
+    await handleStopCompare();
```

Comment:
Since it's just a one-liner, I don't see a reason to create a function just for it?